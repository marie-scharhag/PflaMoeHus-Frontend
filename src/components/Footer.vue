<template>
  <div class="footer">
    <footer>
      <ul>
        <li>
          <h5>Adresse</h5>
          Unter den Eichen 5 <br />
          65195 Wiesbaden
        </li>
        <li>
          <h5>PflaMoeHus GmbH</h5>
          Gesch√§ftsinhaber: Peter Pan <br />
          List-Id: 17171717171717
        </li>
        <li>
          <h5>E-Mail</h5>
          pflamoehus@gmail.com <br />
          Tel.: 01234 56789
        </li>
      </ul>
    </footer>
  </div>
</template>

<style lang="scss" scoped>
*{
    margin: 0;
    padding: 0;
}
.footer {
  clear: both;
}

footer {
    color:white;
    background-color: $color-green;
    height: 10%;
    font-size: 1em;
    
}

footer ul li {
  display: inline-block;
  list-style: none;
  padding: 5% 10%;
  text-align: center;
}
</style>