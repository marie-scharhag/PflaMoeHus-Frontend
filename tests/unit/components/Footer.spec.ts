import Footer from "@/components/Footer.vue"
import { shallowMount } from "@vue/test-utils"

describe('Footer.vue test',() => {
    const cmp = shallowMount(Footer)

    test('snapshot test',() => {
        expect(cmp.html).toMatchSnapshot()
    })
})