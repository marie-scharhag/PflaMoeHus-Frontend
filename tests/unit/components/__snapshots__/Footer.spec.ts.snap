// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue test snapshot test 1`] = `[Function]`;
